src/components/TodoList.js

```diff
@@ -0,0 +1,38 @@
+import keyCodes from "../constant/keyCode.js"
+
+const TodoList = ({$todoList, onToggle, onRemove, onEdit}) => {
+    $todoList.addEventListener("click", (e) => {
+        const idx = parseInt(e.target.closest('li').dataset.index);
+        if(e.target.className === "toggle") {
+            onToggle(idx);
+        }
+        else if(e.target.className === "destroy") {    
+            onRemove(idx);
+        }
+    })
+
+    $todoList.addEventListener("dblclick", (e) => {
+        if(e.target.classList.contains("label")) {
+            e.target.closest('li').classList.toggle("editing");
+        }
+    })
+
+    $todoList.addEventListener("keydown", (e) => {
+        if(e.keyCode === keyCodes.escape) {
+            $todoList.childNodes.forEach(i => {
+                if(i.classList.contains("editing")) {
+                    i.classList.remove("editing")
+                }
```

Comment:
remove() 는 없는 클래스를 제거해도 에러를 일으키지 않는다고 해요
```suggestion
                i.classList.remove("editing")
```

Reply:
오오 네네 그러네요! 감사합니다 불필요..한 코드줄을 줄일 수 있었습니다 :)